core/init/router/router.go

```diff
@@ -69,10 +175,15 @@ func Routers() *gin.Engine {
 	}
 
 	Router.NoRoute(func(c *gin.Context) {
-		c.Writer.WriteHeader(http.StatusOK)
-		_, _ = c.Writer.Write(web.IndexByte)
-		c.Writer.Header().Add("Accept", "text/html")
-		c.Writer.Flush()
+		if checkFrontendPath(c) {
+			toIndexHtml(c)
+			return
+		}
+		if isEntrancePath(c) {
+			toIndexHtml(c)
+			return
+		}
+		handleNoRoute(c)
 	})
 
 	return Router
```

Comment:
The current code seems almost perfect with only some minor improvements. However, here's one area where you may need to optimize based on the current context and requirements:

One thing that could potentially improve performance is removing unnecessary imports at the top of the file since they do not seem to be used anywhere else in the source.

Another improvement suggestion would focus more closely on optimizing specific parts like static content serving using `http.FS`, which can help reduce server load and speed up loading times.

Here’s an optimized version assuming these suggestions are applicable but please let me know if there’s something else I did wrong or missed!